src/search/ir.h

```diff
@@ -356,6 +420,10 @@ struct SortByClause : Node {
   std::unique_ptr<Node> Clone() const override {
     return std::make_unique<SortByClause>(order, Node::MustAs<FieldRef>(field->Clone()));
   }
+
+  std::unique_ptr<FieldRef> GetFieldRef() { return std::move(field); }
+
+  std::unique_ptr<VectorLiteral> GetVectorLiteral() { return std::move(vector); }
```

Comment:
usually "Get" implies that we don't modify the original object, but here `field` and `vector` will be nullptr after moving.

Reply:
I see, will change it to `TakeFieldRef` and `TakeVectorLiteral`.